src/server/game/Battlegrounds/ArenaTeam.cpp

```diff
@@ -899,6 +899,10 @@ void ArenaTeam::SaveToDB()
 
     for (MemberList::const_iterator itr = Members.begin(); itr != Members.end(); ++itr)
     {
+        // Save the effort and go
+        if (itr->WeekGames == 0)
```

Comment:
is this enough to ensure no other variables changed ?
which ArenaTeam values are still needed to be saved even if nothing changed ?